agent_build/requirement-files/testing-requirements.txt

```diff
@@ -48,3 +48,7 @@ more-itertools==8.5.0; python_version >= '3.0'
 more-itertools==5.0.0; python_version <= '2.7'
 distro==1.6.0
 xmltodict==0.13.0; python_version >= '3.0'
+
+paramiko==2.12.0; python_version >= '3.6'
```

Comment:
Do we still need / want to use a separate more minimum requirements file just for the AMI tests (as we do today, iirc)?